src/LLVM_Runtime_Linker.cpp

```diff
@@ -13,10 +13,21 @@ std::unique_ptr<llvm::Module> parse_bitcode_file(llvm::StringRef buf, llvm::LLVM
     llvm::MemoryBufferRef bitcode_buffer = llvm::MemoryBufferRef(buf, id);
 
     auto ret_val = llvm::parseBitcodeFile(bitcode_buffer, *context);
+#if LLVM_VERSION >= 40
+    if (llvm::Error Err = ret_val.takeError()) {
+        std::string Message;
+        handleAllErrors(std::move(Err), [&](llvm::ErrorInfoBase &EIB) {
+            Message = EIB.message();
+        });
+        internal_error << "Could not parse built-in bitcode file " << id
```

Comment:
Might be simpler to put the internal_error inside the lambda, instead of assigning to a local